blog/_posts/2020-4-21-tilt-cloud.md

```diff
@@ -0,0 +1,56 @@
+keywords:
+  - tilt
+---
+
+For Application Developers, production is a datacenter. Infrastructure like CI/CD, Distributed Tracing, and Metrics/SLOs make datacenter apps manageable. But before an AppDev has a PR ready, they run workflows on their laptops. Developer Experience (DevEx) engineers write and manage those workflows, and they can't use datacenter infrastructure. Developer Experience engineers need infrastructure for when production is a coworker's laptop.
+
+Tilt Cloud is a SaaS backend for Tilt that lets you Release, Support, and Measure the Tilt developer experience for your team. If you maintain a Tiltfile that's used by three to three thousand people, Tilt Cloud is for you. The unit of Tilt Cloud organization is a Team. By linking your Tiltfile with a Tilt Cloud Team using the `set_team` function, Tilt Cloud functions as a control plane for the Tilt instances running on each developer's laptop.
+
+Tilt Cloud will help DevEx teams
+* Release improvements fast and safe
+* Support users when they encounter problems
+* Measure the ground truth across all users
+
+We're announcing Tilt Cloud early to work with a handful of design partners. Many of these features are prospective, and we're looking for help prioritizing these features. If you want to upgrade your Tilt experience and shape the direction of Tilt Cloud, [create a Team now](https://cloud.tilt.dev/team/new). We'll reach out to discuss how we can work together.
+
+## DevEx teams need Infrastructure
+Writing the code is just one step to delivering a better experience to the user, whether you're an AppDev and the user is shopping on a smartphone or you're a DevEx and the user is writing code on a laptop. In our conversations with DevEx teams, we've found three areas of infrastructure needs:
+Release
+Support
+Measure
+
+### Release
+When an AppDev runs into foreseeable trouble that wastes their time, they lose faith in the DevEx teams. We're building Tilt Cloud to give DevEx teams confidence about releasing new versions of Tilt.
+
+#### Team Version Visibility
+The first feature of Tilt Cloud is Team Version Visibility. When you open your Tilt Cloud Team page, you'll see the users who've used Tilt with your team recently, along with the version they're using.
+
+#### Team Version Management
+Weekly Tilt releases bring improvements, and occasionally bugs. You'll be able to test new releases of Tilt before they're suggested to every AppDev on your team. All good? Click to promote the new version to your team. Find a problem? Contact us and we'll get a bugfix out before it affects your team.
+
+### Support
+When an AppDev encounters a problem, you want to unblock them quickly. Tilt Cloud will be able to give you more info to help them.
+
+#### Team Snapshots
+Being able to see a user's screen can make problems obvious. Team Snapshots will let you see the same view that flummoxed them. Team Snapshots will only be visible to members of your team, so you can share data securely.
```

Comment:
```suggestion
Being able to see a user's screen can make problems obvious. Team Snapshots will let you see the same view that flummoxed them. Team Snapshots will only be visible to members of your team, so you can share data securely. For now, check out [non-team snapshots](https://docs.tilt.dev/snapshots.html). 
```

Reply:
I think it might be confusing to have that and try to describe each; if it's okay I'm going to leave that out.

Reply:
Sure. Resolving. No change.